requirements.txt

```diff
@@ -26,32 +25,24 @@ iniconfig==1.1.1
 isort==5.12.0
 jmespath==0.10.0
 jsonschema==4.5.1
-lz4==3.1.3
 mccabe==0.6.1
 mmh3==3.0.0
 multidict==6.0.2
 mypy==0.950
 mypy-extensions==0.4.3
-neo-mamba==0.10.0
-neo3crypto==0.2.1
-neo3vm==0.9.0
-neo3vm-stubs==0.9.0
-neofs-testlib==0.9.0
+neo-mamba==1.0.0
```

Comment:
it's not required here since it already in testlib 
https://github.com/nspcc-dev/neofs-testlib/blob/master/pyproject.toml#L22